### ISSUE TITLE
Maybe type error?

### ISSUE THREAD

**Issue by nightire** [alexlafroscia/qunit-wait-for#8]
https://github.com/alexlafroscia/qunit-wait-for/blob/20a350e8da789637d61ad0d6c216a5b443eae83b/src/index.ts#L52-L55

I believe `wait-for` returns a `Promise` not a function that returns a `Promise`, maybe there's an error here.

**Comment by nightire** [comment]
Unfortunately, even I fixed it locally, I still face a type error which I don't know what's going on:

![](https://media.discordapp.net/attachments/484421406659182603/702093400526028830/unknown.png)

I asked this in Discord channel as well: https://discordapp.com/channels/480462759797063690/484421406659182603/702093402585432074

**Comment by alexlafroscia** [comment]
Great catch! I'll get that fixed right up.

I had noticed something "off" about the `assert.waitFor` typing as well, as my editor wasn't picking it up correctly. Turns out it was the fact that my test file was explicitly importing `QUnit`, which does not work with the `@types/qunit` package.

Once I removed that, `assert` got the right type, so `assert.waitFor` started showing me the same unexpected type that you saw.

This should be fixed up by #11
